021-changelog-upgrade.md

```diff
@@ -33,7 +33,7 @@ For each version there are number of lines listing the changes. "Bug" refers to
 In the changelog file lines should be ordered as New, Chg, Enh, Bug.
 
 "#42" above is the number of issue or pull request corresponding to the change. "author1" is the GitHub nickname of the
-code author. "author2" is additional author.
+code author. "author2" is additional author. Author nickname MUST be prefixed by `@`.
```

Comment:
```suggestion
code author. "author2" is additional author. An author's nickname *must* be prefixed with `@`.
```

Reply:
Keywords "must", "should" and etc. are usually written in uppercase. For example, [PSR documents](https://www.php-fig.org/psr/psr-11/) or [Yii docs](https://github.com/yiisoft/docs/blob/master/000-packages.md#php-versions-support).